submission/proxy_test.go

```diff
@@ -165,3 +166,18 @@ Init:
 		}
 	}
 }
+
+func TestASN1MarshalSCT(t *testing.T) {
+	rawSCT := testdata.TestCertProof
+	var sct ct.SignedCertificateTimestamp
+	_, err := tls.Unmarshal(rawSCT, &sct)
+	if err != nil {
+		t.Fatalf("failed to tls-unmarshal test certificate proof: %s", err)
+	}
+	assignedSCT := AssignedSCT{LogURL: "ct.googleapis.com/racketeer/", SCT: &sct}
+	assignedSCTs := []*AssignedSCT{&assignedSCT}
+	_, err = ASN1MarshalSCTs(assignedSCTs)
```

Comment:
This should probably be testing that the result matches the original input. Maybe that's covered elsewhere.

Reply:
Added bytes to compare against.